packages/fury-adapter-oas3-parser/lib/parser/oas/parseSecuritySchemeObject.js

```diff
@@ -57,8 +57,20 @@ function validateApiKeyScheme(context, securityScheme) {
 function validateHttpScheme(context, securityScheme) {
   const { namespace } = context;
 
+  const schemes = ['bearer', 'basic'];
+  const isValidScheme = R.anyPass(R.map(hasValue, schemes));
+  const createInvalidSchemeWarning = scheme => createWarning(
+    namespace,
+    `'${name}' 'http' contains unsupported scheme '${scheme.value.toValue()}', supported schemes ${schemes.join(', ')}`,
+    scheme.value
+  );
+
+  const parseScheme = pipeParseResult(namespace,
+    parseString(context, name, false),
+    R.unless(isValidScheme, createInvalidSchemeWarning));
+
   const parseMember = R.cond([
-    [hasKey('scheme'), parseString(context, name, false)],
+    [hasKey('scheme'), parseScheme],
 
     [innerPassThrough, e => e],
```

Comment:
Out of curiosity: could we use [`R.identity`](https://ramdajs.com/docs/#identity) here?

Reply:
Yes! It's there for consistency as this was done throughout the code base. Could do with changing them all.